deploy/06_deploy_rewards_pool.js

```diff
@@ -0,0 +1,15 @@
+module.exports = async ({ getNamedAccounts, deployments }) => {
+  const { deploy } = deployments
+  const { deployer, rewardManager } = await getNamedAccounts()
+  const KeepToken = await deployments.get("KeepToken")
+  const AssetPool = await deployments.get("AssetPool")
+
+  await deploy("RewardsPool", {
```

Comment:
Do we need a separate script to deploy `RewardsPool`? An instance of `RewardsPoll` is already deployed as part of the `AssetPool` constructor
https://github.com/keep-network/coverage-pools/blob/34c3d2f6bdb1c608a755dae07f39d29c52297126/contracts/AssetPool.sol#L125

Reply:
Let's say it's a workaround to get the `RewardsPool` contract abi in the `deployments` dir. The hardhat splits the compiled and deployed contracts artifacts between different dirs. In this repo hardhat saves compiled artifacts to `./build` but deployed artifacts to `./deployments/<network>` and we create a symbolic link only to the `deployments/<network>` dir.

Reply:
So the `RewardsPool` contract will be deployed twice, first in the `AssetPool` constructor and second in the `06_deploy_rewards_pool.js`. But we will use the address from `AssetPool` instead of the deployed artifact. This is very likely to cause a confusion.

What if we remove the `06_deploy_rewards_pool.js` script and expect the contract to be deployed inside the `AssetPool` constructor. We will obtain the address easily with a call to `AssetPool.rewardsPool`. For the `RewardsPool` ABI we will have to use compilation artifacts from the `./build` directory.

Reply:
Please see b1cdc5e.